lib/scaffold.dart

```diff
@@ -508,9 +508,9 @@ class BackdropScaffoldState extends State<BackdropScaffold>
           behavior: HitTestBehavior.opaque,
           child: Column(
             children: <Widget>[
-              // if subHeaderAlwaysActive then do not apply inactiveOverlayColor for area with _headerHeight
+              // if subHeaderAlwaysActive then do not apply frontLayerScrim for area with _subHeaderHeight
               widget.subHeader != null && widget.subHeaderAlwaysActive
-                  ? Container(height: _headerHeight)
+                  ? Container(height: _subHeaderHeight)
```

Comment:
Not related to this PR, but seemed like a mistake that interacts oddly with `_headerHeight`.  

With this change, `subHeaderAlwaysActive = true` gives

![Screenshot from 2021-02-04 22-27-41](https://user-images.githubusercontent.com/705804/106985933-5a8dbd00-6738-11eb-974f-be6326ac2219.png)

`subHeaderAlwaysActive = false` applies the scrim (inactiveOverlay) to the entire front.

![Screenshot from 2021-02-04 22-28-17](https://user-images.githubusercontent.com/705804/106985940-5cf01700-6738-11eb-85a0-990ca4b58c39.png)

Have I misunderstood the intent?  If the user wants no overlay/scrim, they can pass color `transparent` or `null`.

Reply:
It looks like a mistake, i dont know how it landed! Did you test the fix properly? It isnt breaking anything else right?

Reply:
Sorry, it took me late to respond, I most work on it in weekends.

Reply:
resolving this.